utils.pm

```diff
@@ -432,6 +432,7 @@ sub run_ltp
 		push(@{$result->{'log'}}, @log);
 		$result->{'runtime'} += $test_end_time - $test_start_time;
 		$result->{'runs'} += 1;
+		push @{$result->{'retval'}}, (defined($ret) ? $ret : 'undefined');
```

Comment:
Shouldn't we initialize the $result->{'retval'} first time we attempt to push there?

Reply:
done

Reply:
Not sure if empty string would be better than `undefined` (which evaluates as true).

Reply:
@pevik I guess that anything that is not number will do.

This looks fine now, but I guess that you do miss the second part, which is the to make use of this in the openqa_filter.

Reply:
> Not sure if empty string would be better than `undefined` (which evaluates as true).

The value will not be evalueated as true/false. It'll be matched against a regex written in `ltp_known_issues.json`. When we want to whitelist a failure where the test times out, the regex in question will be "retval": "^undefined$"`.

Reply:
> @pevik I guess that anything that is not number will do.
> 
> This looks fine now, but I guess that you do miss the second part, which is the to make use of this in the openqa_filter.

done